src/service/vsync-impl.js

```diff
@@ -128,6 +130,21 @@ export class Vsync {
       // per-doc visibility when necessary.
       this.docState_.onVisibilityChanged(boundOnVisibilityChanged);
     }
+
+    /** @private {?Element} */
```

Comment:
Please refactor this CL to isolate the jank rate specific functionality into a file or at least functions.